docs/modeling-your-data/modeling-your-data-with-dbt/dbt-configuration/web/index.md

```diff
@@ -110,6 +110,7 @@ Redshift and Postgres use a [shredded](/docs/pipeline-components-and-application
 | `yauaa_context`       | `nl_basjes_yauaa_context_1`                            |
 | `consent_cmp_visible` | `com_snowplowanalytics_snowplow_cmp_visible_1`         |
 | `consent_preferences` | `com_snowplowanalytics_snowplow_consent_preferences_1` |
+| `cwv_context`         | `com_snowplowanalytics_snowplow_web_vitals_1`          |
```

Comment:
these are events

Reply:
FYI as it is only changed in unified

Reply:
Yeah it's not ideal but didn't want to make a breaking change, that can be a v1 problem :D

Reply:
sure